scripts/Test-FaultInjection.ps1

```diff
@@ -22,6 +22,16 @@ Set-Content -Path ($TestProgram + ".fault.log") ""
 
 $iteration = 0
 
+$timer = New-Object System.Timers.Timer
+# Set timer for 5 minutes
+$timer.Interval = 300000
```

Comment:
Is this time value even for fault_injection_full?

Reply:
It shouldn't. Each iteration should take less than 5 minutes unless it's stuck.

Reply:
Thanks for clarifying. The reason of ask is, we have more APIs added with fault injection.